### DISCUSSION TITLE
Parameters file?

### DISCUSSION THREAD

**Discussion by Munchic** [vdemichev/DiaNN#545] [Q&A]
Hi Vadim,

I was wondering how to save a configuration file storing all search parameters chosen in the GUI to then be used in the CLI version? Thank you!

Munchic

**Comment by vdemichev** [comment]
Hi Munchic,

When you run DIA-NN from the GUI, the GUI prints the exact command used in the log window. Can copy that and use to invoke the command like tool.

Best,
Vadim

**Reply by Munchic** [reply]
Thank you!
